src/pages/MainQuizScreen.tsx

```diff
@@ -74,8 +83,27 @@ const isCorrectAnswer = (
 
 const MainQuizScreen: React.FC = () => {
   const [currentQuestionId, setCurrentQuestionId] = useState<number>(1);
-  const [feedback, setFeedback] = useState<string>('');
+  const [feedback, setFeedback] = useState<string>(FEEDBACK.default);
   const [isQuizCompleted, setIsQuizCompleted] = useState<boolean>(false);
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
+
+  const handleOpenModal = () => {
+    setIsModalOpen(true);
+  };
+
+  const handleCloseModal = () => {
+    setIsModalOpen(false);
+  };
```

Comment:
let's repalce these 2 with one `handleToggleOpenModal` function, since by default you have a `false` value for `isModalOpen`
```ts
const handleToggleOpenModal = () => {
  setIsModalOpen(value => !value)
}
```